src/page_objects/page.js

```diff
@@ -397,7 +398,7 @@ module.exports = {
             for (var key in list) {
               if (key == "Name") {
                 const selector = {
-                  selector: `.//tr[@class='is-new']/td[@data-label='Name']`,
```

Comment:
This works in a test run of TAF on github actions for BI-807 Scenario Outline: Program Location Management page but fails when there are multiple locations on my local since the new entry isn't necessarily the first entry. Since there will be multiple scenarios adding program locations once BI-1313 is implemented, should there be logic to iterate through the entries in the list to find the new item, and should that be part of this card or another if so?

Reply:
This is a limitation with the current implementation.
Since we have removed an class "is-new", we can't find the newly created row especially when the table supports non-unique names.
We can combine other fields to identify the unique entry. I don't know what they are for Program Location.

Reply:
@cvacalares That's a fair point.  Do object that get created for tests in TAF generate a random (and most likely unique) name?  If so, I think that would allow searching for a specific name to be a feasible option.

Reply:
@timparsons 
We have been using random string by replacing "*" with a timestamp or some random string.
args1.replace("*", this.parameters.timeStamp)
args1.replace("*", generateRandomAlphaString(4))

Reply:
@cvacalares Ok, so do you think it'd be reasonable to search through the records in a table for that specific string?